exampleservice/CustomerService/Utils/Password.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace exampleservice.CustomerService.Utils
+{
+    public class Password
```

Comment:
could be `public static class` right? Also the name `Password` is quite misleading because I would assume it to be a data-type and no static utility class. Suggestion: Move to the base Namespace and rename to `PasswordUtils`.

Reply:
always a valid discussion :D personally in my projects i use a package "utils" with classes named "XxxUtils"

Reply:
added `static` position/name is not that relevant for now :)